src/main/java/com/minelittlepony/model/armour/ModelZombiePonyArmor.java

```diff
@@ -0,0 +1,29 @@
+package com.minelittlepony.model.armour;
+
+import com.minelittlepony.render.AbstractPonyRenderer;
+
+public class ModelZombiePonyArmor extends ModelPonyArmor {
+    // Copied from ModelZombiePony
```

Comment:
We should just have one armor model for all the ponies. That can be done later.

Reply:
Yeah, I have that done in a separate branch.